lib/Doctrine/DBAL/Platforms/DB2Platform.php

```diff
@@ -878,4 +881,57 @@ protected function getReservedKeywordsClass()
     {
         return Keywords\DB2Keywords::class;
     }
+    
+    /**
+     * Prepare ORDER BY string for OVER() if applicable
+     * 
+     * @param  string $query
+     * 
+     * @return string
+     */
+    private function getOrderByForOver($query)
+    {
+        //preset empty array
+        $orderByArray = [];
+
+        // remove extra white space
+        $query = preg_replace('/\s+/', ' ', $query);
```

Comment:
This can (or even should) be replaced with a proper `preg_split()` expression once the `ORDER BY` part is detected. No need to process the entire query.

Reply:
In here I try to eliminate dead whitespace.
However, if it bugs you, I will reconsider.

Reply:
Also, it simplifies the processing a little bit along the line.